surveys/2024-annual-survey/translations/fr.md

```diff
@@ -276,140 +276,140 @@ Rows:
 Columns:
 
 - Je suis d’accord
-- Je ne suis pas d'accord
+- Je ne suis pas d’accord
 
-### 
+### Quelle est votre opinion sur la vitesse d’évolution de Rust ?
 
 Type: select one
 
-- 
-- 
-- 
-- 
-- 
-- Other
+- Rust est déjà trop complexe, il ne devrait pas ajouter ou stabiliser des fonctionnalités plus importantes
+- Rust évolue trop rapidement, j’aimerais que le rythme de développement ralentisse
+- Je suis satisfait du rythme actuel de développement
+- Rust évolue trop lentement, j’aimerais que des fonctionnalités soient ajoutées ou stabilisées plus rapidement
+- Je ne sais pas ou je m&#39;en fiche
+- Autre
 
-### Quelles fonctionnalités non implémentées ou (uniquement nightly) souhaitez-vous stabiliser ?
+### Quelles fonctionnalités non implémentées (ou uniquement nightly) souhaitez-vous stabiliser ?
 
 Type: matrix
 
 Rows:
 
-- Spécialisation
-- Générateurs/coroutines
-- Closures asynchrones
-- 
--  Si/pendant que les chaînes if/while let
-- Blocs try
-- Type never
-- Traits améliorés (alias de trait, limites implicites, valeurs par défaut de type associées)
-- 
-- 
-- 
-- 
-- 
-- 
-- 
-- 
-- Trait d&#39;allocation, meilleure gestion du MOO
-- IBC stable
-- 
+- [Spécialisation](https://github.com/rust-lang/rust/issues/31844)
+- [Générateurs/coroutines](https://github.com/rust-lang/rust/issues/43122)
+- [Closures asynchrones](https://github.com/rust-lang/rust/issues/62290)
+- [Générateurs/coroutines asynchrones](https://github.com/rust-lang/rust/pull/118420)
+- [If/while let chaînés](https://github.com/rust-lang/rust/issues/53667)
+- [Blocs try](https://github.com/rust-lang/rust/issues/31436)
+- [Type never](https://github.com/rust-lang/rust/issues/35121)
+- [Alias de traits](https://github.com/rust-lang/rfcs/blob/master/text/1733-trait-alias.md)
+- [Trait d&#39;implémentation d&#39;alias de type (TAIT)](https://rust-lang.github.io/rfcs/2515-type_alias_impl_trait.html)
+- [Valeurs par défaut des types associés](https://rust-lang.github.io/rfcs/2532-associated-type-defaults.html)
+- Expressions constantes génériques
+- [Méthodes de trait constant](https://github.com/rust-lang/rust/issues/67792)
+- Réflexion au moment de la compilation
+- Génériques variadiques
+- [Types de soi arbitraires](https://github.com/rust-lang/rfcs/blob/master/text/3519-arbitrary-self-types-v2.md)
+- [Types de variantes d’énumération](https://github.com/rust-lang/lang-team/issues/122)
+- [Trait Allocator et meilleure gestion des épuisements de mémoire](https://github.com/rust-lang/rust/issues/32838)
+- [ABI stable](https://github.com/rust-lang/rust/issues/111423)
+- [SIMD portable](https://github.com/rust-lang/portable-simd)
```

Comment:
I am honestly in awe by how the French translate really **everything** :-D

Reply:
I might have been taken over by a bit of Quebec tbh